app/views/search/index.html.erb

```diff
@@ -1,15 +1,17 @@
-<div class="page-header">
+<div class="page-header search">
   <h1>
-    <%= pluralize(@results.count, "result") %> for "<%= @query %>"
+    <span><%= pluralize(@results.count, "result") %></span> for "<%= @query %>"
   </h1>
 </div>
 
 <% if @results.any? %>
   <ul class="ui-search-list">
     <% @results.each do |result| %>
-      <li>
-        <%= render result.searchable_type.downcase, { model: result.searchable } %>
-      </li>
+      <%= render result.searchable_type.downcase, { model: result.searchable } %>
     <% end %>
   </ul>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<script type="text/javascript">
+  var user_project_ids = <%= MultiJson.dump(current_user.projects.pluck(:id)) %>;
```

Comment:
Would be nicer to limit the number of global variables we expose in JS. Since in `application.js` we setup a global `App` variable might be good to make it have an `App.data = {}` then we can just chain all data to it like:

```js
App.data.user_project_ids = <%== MultiJson.dump(current_user.project_ids) %>;
```